app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<h1>Task List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <th class="center">
+      Mark Done
+    </th>
+    <th class="center">
+      Details
+    </th>
+    <th class="center">
+      Delete
+    </th>
+  </tr>
+  <tbody>
+    <% @all_tasks.each do |task| %>
+      <% if task.completed_date.present? %>
+        <% @task_css = "completed" %>
```

Comment:
I would add the method I described above to the Task model (because it's logic about the model).

I would then add a custom rails helper as `tasks_helper.rb`

```ruby
def task_completed_css_class(task)
  task.completed? ? "completed" : "incomplete"
end
```